proco/schools/loaders/brasil_loader.py

```diff
@@ -37,6 +37,15 @@ def save_school(self, school_data):
                 },
             )
 
+            date = timezone.now().date()
+            SchoolWeeklyStatus.objects.update_or_create(
+                school=school, week=date.isocalendar()[1], year=date.isocalendar()[0],
+                defaults={
+                    'computer_lab': bool(int(float(school_data.get('QT_COMP_ALUNO', 0)))),
+                    'num_computers': int(float(school_data.get('QT_COMPUTADOR', 0))),
```

Comment:
there are more fields that we can use
TP_LOCALIZACAO=school.environment
TIPO_TECNOLOGIA=connectivity_type